### DISCUSSION TITLE
Значения по CodeableConcept в Coverage.Type для полисов ОМС и ДМС

### DISCUSSION THREAD

**Discussion by abadretdinova** [fhir-ru/core#98] [element]
Необходимо определить, какие коды допустимо применять для обозначения полисов ОМС и ДМС в Coverage.

В Coverage.Type можно указывать значения по CodeableConcept. Пока на примете следующие значения:
- для ОМС: MANDPOL, PUBLICPOL
- для ДМС: EHCPOL, EWB

Coverage описан тут: http://hl7.org/fhir/coverage.html
Коды, применяемые для Coverage.Type тут: http://hl7.org/fhir/valueset-coverage-type.html

**Comment by eugenekogan-spb** [comment]
Максим Алексеев:
там CodeableConcept. в таких случаях я предпочитаю не искать счастья в FHIR valueset, а использовать сторонние справочники, в идеале федеральные. в данном случае я бы взял https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.11.1039

**Comment by AndreyAlabin** [comment]
EHCPOL - на мой взгляд, наиболее подходящая под ДМС (voluntary health insurance)

PUBLICPOL, MANDPOL, HIP - возможно какое-то из этих трех полей возможно адаптировать, но ни одно из них полностью не соответствует ОМС (compulsory health insurance). PUBLICPOL - как я понимаю, система с бюджетным финансированием, HIP - модель со страховыми планами, возмещаюми затраты в пределах определённой суммы, что тоже не случай с ОМС. В таком случае наиболее приемлимым для ОМС является MANDPOL так как ОМС это тоже программа государственных гарантий бесплатного оказания гражданам медицинской помощи. Форма реализации и система финансирования (через страхование или бюджетное финансирование уже вопрос второй).

**Comment by eugenekogan-spb** [comment]
От Андрея Алабина - из чата FHIR-RU

Коллеги, день добрый! В продолжение вчерашней темы про ОМС/ДМС задал вопрос в FHIR на org, получил ответ. Выкладываю сюда. В целом, получается, что, до принятия решения по этому вопросу в RU Core, можно пользоваться имеющимися кодами существующей структуры формы, их смысловое наполнение позволяет. 
<img width="866" alt="image" src="https://user-images.githubusercontent.com/62525329/161252867-7bfe9515-1d2b-47c3-adfd-dced2abeed55.png">
